_layouts/home.html

```diff
@@ -0,0 +1,227 @@
+                    <li><a href="#what" class="btn btn-white">Learn More</a></li>
+                </ul>
+            </div>
+            <div class="col-lg-5 col-md-6 mr-auto">
+                <div class="inner-img">
+                    <img src="{{ '/assets/img/intro.svg' | relative_url }}">
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container text-center why-vac" id="why">
+        <div class="row">
+            <div class="col-lg-9 col-md-12 ml-auto mr-auto">
+                <h2 class="section-title bar-center"><b>WHY Vac</b></h2>
+                <p>Privacy is a fundamental human right. The ability to message without fear of censorship or prosecution must be maintained. The ability to conduct transactions with whom you want, and only whom you want is a requirement for free communication. This is why we build vacp2p</p>
+                <hr>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="log" id="research-log">
```

Comment:
This can possibly take up more visual space, e.g. compare space taken in current design vs https://vac.dev - I'd consider bringing this above the fold

Reply:
I made it a little bit bigger.